### DISCUSSION TITLE
r68s docker会根目录会自动切换为overlayfs:/overlay

### DISCUSSION THREAD

**Discussion by EasierLu** [coolsnowwolf/lede#9533] [Bugs / 问题反馈]
无法使用/dev/mmcblk0p7
重启后会自动将overlayfs:/overlay挂载为docker根目录

**Comment by EasierLu** [comment]
已解决，设置问题

**Reply by ghost** [reply]
兄弟 你是怎么解决的？我看默认升级docker固件 也是将overlayfs:/overlay挂载为docker根目录
